doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -1965,6 +1965,7 @@ The whole DPDK configuration resides in the `dpdk:` node. This node encapsulates
     dpdk:
       eal-params:
         proc-type: primary
+        allow: ["0000:3b:00.0"]
```

Comment:
I think it would be useful for users if we would add a second PCIe address just to make it more clear how it works. 
So the parameter would look like this:
`allow: ["0000:3b:00.0", "0000:3b:00.1"]`

Reply:
I am very sorry for not replying to the message in the first time. I have updated the code as suggested. Your  @lukashino enthusiastic suggestions are the driving force for me to continuously improve the code, and it also makes me feel the high standards required as a committer. I will continue to track relevant updates and contribute my own strength,  Thanks again.

Reply:
That's very good to hear @zhiyuan-liao! Ok, the documentation is finished, now we can work on the commits.
As changes are tightly coupled and close-related, we can put the changes into a single commit. 

So, please:
1. Create a new branch `dpdk_params_list_v3`
2. Squash all three commits into one
3. Reword the commit message. It can look like this:
```
dpdk: support multiple same EAL arguments

DPDK apps can specify multiple arguments of the same
type. YAML format only allows unique keys within a single
node. This commit adds support for multiple EAL arguments
of the same type to be used within suricata.yaml.

Ticket: #5964
```
4. Submit the newly created branch as a new Pull Request, where in the description you reference this Pull Request (in the description of the new PR mention `Follow-up of #8947`).
5. Close this PR with a comment e.g. `Continues in #ID_NEW_PR`

I am always eager to see and help new contributors so no worries!